composer.json

```diff
@@ -1,56 +1,46 @@
 {
-  "name": "godaddy/go",
-  "type": "wordpress-theme",
-  "authors": [
-    {
-      "name": "GoDaddy",
-      "homepage": "http://www.godaddy.com",
-      "role": "Developer"
-    }
-  ],
-  "autoload": {
-    "psr-4": {
-      "Go\\": "includes/classes/"
-    }
-  },
-  "repositories": [
-    {
-        "type":"composer",
-        "url":"https://wpackagist.org"
-    }
-  ],
-  "extra": {
-    "installer-paths": {
-      "../../plugins/{$name}/": [
-        "type:wordpress-plugin"
-      ]
-    }
-  },
-  "config": {
-    "platform": {
-      "php": "7.4"
+    "name": "godaddy/go",
+    "type": "wordpress-theme",
+    "authors": [
+        {
+            "name": "GoDaddy",
+            "homepage": "http://www.godaddy.com",
+            "role": "Developer"
+        }
+    ],
+    "autoload": {
+        "psr-4": {
+            "Go\\": "includes/classes/"
+        }
+    },
+    "repositories": [
+        {
+            "type": "composer",
+            "url": "https://wpackagist.org"
+        }
+    ],
+    "config": {
+        "platform": {
+            "php": "7.4"
+        },
```

Comment:
Given that 7.4 is EOL and is in security maintenance only, should we uplift it to 8.2 here? I know we have unit test pipelines set up for 8.2 and 7.4. I don't know how much it matters here.

Reply:
I kept this as is for now. I think if we were to bump it we'd bump to 8.1 but no discussion has happened and with the removal of so much PHP code it wouldn't make a real differnce.